playbook.py

```diff
@@ -0,0 +1,486 @@
+"""
+"""
+
+import phantom.rules as phantom
+import json
+from datetime import datetime, timedelta
+def on_start(container):
+    phantom.debug('on_start() called')
+
+    # call 'filter_1' block
+    filter_1(container=container)
+
+    return
+
+def get_alert_details_1(action=None, success=None, container=None, results=None, handle=None, filtered_artifacts=None, filtered_results=None, custom_function=None, **kwargs):
```

Comment:
The styler will love this!

I actually am not sure that we are supposed to be committing this here. Hopefully we can!

Reply:
Is there a way we can make `black` ignore this file, since it's autogenerated

Reply:
Good call, I'll look into that